### DISCUSSION TITLE
Difference between external-secrets and kubernetes-external-secrets

### DISCUSSION THREAD

**Discussion by cikupin** [external-secrets/kubernetes-external-secrets#869] [Q&A]
Hi all, what are the differences between [external-secrets](https://github.com/external-secrets/external-secrets) and [kubernetes-external-secrets](https://github.com/external-secrets/kubernetes-external-secrets)?

I see that both of them have functionality to integrate secret from GCP or AWS to kubernetes.

**Comment by Flydiverny** [comment]
#864 if you haven't seen that one

**Reply by cikupin** [reply]
ah thank you
